Lib/asyncio/base_events.py

```diff
@@ -200,6 +200,17 @@ def _set_nodelay(sock):
         pass
 
 
+if hasattr(socket, 'TCP_QUICKACK'):
+    def _set_quickack(sock, val):
```

Comment:
It's unused in asyncio. What's the purpose of this function? I suggest to remove it.

Reply:
It supports the test case and mirrors what other socket flags tests were doing.  Click the up arrow and expand your compare upwards and you'll see I copied the TCP_NODELAY test.

Reply:
Refactoring all the socket related TCs is a different PR.

Reply:
If it's only used by tests, but please move the function to tests.

_set_nodelay() is different, it's used by Lib/asyncio/selector_events.py and Lib/asyncio/proactor_events.py.

Reply:
I didn't realize, thank you for pointing that out, will move